src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
I think `Optional<String>` is indeed the more consistent way to go. Also has the advantage that the methods of the optional can be used (map, ifPresent etc).